packages/theme/src/baseTheme.ts

```diff
@@ -57,9 +59,17 @@ export function createThemeOptions(
       },
     },
     typography: {
+      htmlFontSize,
       fontFamily,
+      h6: {
+        fontWeight: 700,
+        fontSize: 20,
+        marginBottom: 2,
+      },
       h5: {
         fontWeight: 700,
+        fontSize: 24,
+        marginBottom: 4,
```

Comment:
Nice! I've just a question about the margin and fontWeights. Do we intentionally change this from the base material ui styles? And why? I'm looking at the default styles of the typography component -> https://v3.mui.com/style/typography/#typography

Reply:
Good question, I have no context because we changed this in the Backstage theme, maybe it was a design demand in the past, let's clarify with the @backstage/maintainers team.

Reply:
Yeah I have no idea either :) @Rugvip do you remember any history of this? Obviously the old ones below do this too